Makefile

```diff
@@ -0,0 +1,25 @@
+#
+# Makefile to run the business analytics software using metabase
+#
+
+#runs the docker-compose.yml showing logs
+up :
+	@echo "Starting Riff Business Analytics.";
+	@docker-compose up;
+	@echo "Processes Complete. Metabase running on port 3000.";
+
+#runs the docker-compose.yml detached
+up-detach :
+	@echo "Starting Riff Business Analytics.";
+	@docker-compose up --detach;
+	@echo "Processes Complete. Metabase running on port 3000.";
+
+#stops and removes containers
+down :
+	@docker-compose down
+
+#runs the script to restore database backups for metabase to run queries on
+restore :
+	@echo "Restoring mysql database backups from 'database-backups' folder.";
+	@bash bin/restore-database-backups.sh;
```

Comment:
> Using the `Makefile` in riff-docker as an example, having the default target be help which tells you what the useful targets in the Makefile are and what they do is worth doing.
> 
> I still have to actually try to run the environment and see if I can create any charts, following the documentation. I may have more comments after I do that.

Added help as default target.

Reply:
Without explicitly using @bash, I get a permission denied error...Not exactly sure what the root issue is.

Reply:
Ah, did you try running this script from your terminal directly?
I suspect you haven't made the file executable (`chmod +x bin/restore-database-backus.sh`)

Reply:
Also, it made the command silent I guess, which I thought made the console look cleaner.

Reply:
🤖 Beep boop. This is the totally-automated-I-am-definitely-not-a-human @bash bot saying hello 👋. Beeb boop 🤖.

Reply:
+1 on permissions. Could also use 1>/dev/null if command silence is desired, to preserve stderr output?

Reply:
Ok I will try that. Command silence just seemed sleeker to me, but I don't think it's a priority.

Reply:
just a note. command silence from the command and using a subshell are 2 different things.
When you say command silence what exactly do you mean, and at least initially it would seem to me that more feedback is better in case things go wrong in ways you didn't anticipate. Squelching the output seems better when you gracefully handle things going wrong. I just don't think this is at that point yet.

Reply:
Alright that does make sense. I will leave it for now.